src/analysis/complex/circle.lean

```diff
@@ -97,3 +99,76 @@ def exp_map_circle_hom : ℝ →+ (additive circle) :=
   map_zero' := by { rw exp_map_circle, convert of_mul_one, simp },
   map_add' := λ x y, show exp_map_circle (x + y) = (exp_map_circle x) * (exp_map_circle y),
     from subtype.ext $ by simp [exp_map_circle, exp_add, add_mul] }
+
```

Comment:
```suggestion
attritbute [simps] exp_map_circle

lemma exp_map_circle_hom_ker : exp_map_circle_hom.ker = add_subgroup.gmultiples (2 * π) :=
begin
  ext k,
  have a0 : (0 : additive circle) = (1 : circle) := rfl,
  have iI : ∀ (x y : ℂ), x * I = y * I ↔ x = y :=
    λ _ _, ⟨mul_right_cancel' I_ne_zero, λ h, congr_arg _ h⟩,
  suffices : (∃ (n : ℤ), (k : ℂ) = n * 2 * π) ↔ ∃ (n : ℤ), (n : ℝ) * 2 * π = k,
  { simpa [add_subgroup.mem_gmultiples_iff, add_monoid_hom.mem_ker, subtype.ext_iff,
           a0, exp_eq_one_iff, ←mul_assoc, iI] },
  simp_rw @eq_comm _ _ k,
  norm_cast
end

```
Really, the `@[simps]` tag should go on the `def` above. Then you can use this statement further in your file.